controller_interface/include/controller_interface/controller_interface_base.hpp

```diff
@@ -187,6 +190,7 @@ class ControllerInterfaceBase : public rclcpp_lifecycle::node_interfaces::Lifecy
   std::vector<hardware_interface::LoanedCommandInterface> command_interfaces_;
   std::vector<hardware_interface::LoanedStateInterface> state_interfaces_;
   unsigned int update_rate_ = 0;
+  bool is_async_ = false;
```

Comment:
This just reminded me that it'd be really nice to have the ros2_control CLI report if a controller is async, similarly to how it already report chainability and whether it's currently chained